File: C#/ExcelSheetColumnNumber.cs
public class Solution171 {
    public int TitleToNumber(string columnTitle) {
    
       int ans = 0;

       foreach(char c in columnTitle)
       {
           int d = c - 'A' + 1;

           ans = ans * 26 + d;
       }
        
        return ans;
    }
}

File: Python3/Excel Sheet Column Number.py
class Solution:
    def titleToNumber(self, columnTitle: str) -> int:
        total = 0
        count = 0
        for char in columnTitle[::-1]:
            total = total + pow(26, count) * (ord(char) - 64)
            count = count + 1
        
        return total